portals/publisher/src/main/webapp/site/public/locales/en.json

```
@@ -8699,24 +8795,302 @@
+      "value": "Something went wrong while downloading the policy"
+    }
+  ],
+  "Global.Details.Policies.AttachedPolicyCard.commonPolicy.download.error": [
+    {
+      "type": 0,
+      "value": "Something went wrong while downloading the policy"
+    }
+  ],
+  "Global.Details.Policies.AttachedPolicyForm.General.cancel": [
+    {
+      "type": 0,
+      "value": "Cancel"
+    }
+  ],
+  "Global.Details.Policies.AttachedPolicyForm.General.description.title": [
+    {
+      "type": 0,
+      "value": "Description"
+    }
+  ],
+  "Global.Details.Policies.AttachedPolicyForm.General.description.value.not.provided": [
+    {
+      "type": 0,
+      "value": "Oops! Looks like this policy does not have a description"
+    }
+  ],
+  "Global.Details.Policies.AttachedPolicyForm.General.description.value.provided": [
+    {
+      "type": 1,
+      "value": "description"
+    }
+  ],
+  "Global.Details.Policies.AttachedPolicyForm.General.reset": [
+    {
+      "type": 0,
+      "value": "Reset"
+    }
+  ],
+  "Global.Details.Policies.AttachedPolicyForm.General.save": [
+    {
+      "type": 0,
+      "value": "Save"
+    }
+  ],
+  "Global.Details.Policies.AttachedPolicyForm.General.saving": [
+    {
+      "type": 0,
+      "value": "Saving"
+    }
+  ],
+  "Global.Details.Policies.DraggablePolicyCard.policy.view": [
+    {
+      "type": 0,
+      "value": "View"
+    }
+  ],
+  "Global.Details.Policies.PolicyConfigurationEditDrawer.title": [
+    {
+      "type": 0,
+      "value": "Configure "
+    },
+    {
+      "type": 1,
+      "value": "policy"
+    }
+  ],
+  "Global.Details.Policies.PolicyConfiguringDrawer.title": [
+    {
+      "type": 0,
+      "value": "Configure "
+    },
+    {
+      "type": 1,
+      "value": "policy"
+    }
+  ],
+  "Global.Details.Policies.PolicyList.title": [
+    {
+      "type": 0,
+      "value": "Policy List"
+    }
+  ],
+  "Global.Details.Policies.SaveOperationPolicies.save": [
+    {
+      "type": 0,
+      "value": "Save"
+    }
+  ],
+  "Global.Details.Policies.SaveOperationPolicies.update": [
+    {
+      "type": 0,
+      "value": "Update"
+    }
+  ],
+  "Global.Policies": [
+    {
+      "type": 0,
+      "value": "Global Policies"
+    }
+  ],
+  "Global.Policy.Listing.Table.Header.Name": [
+    {
+      "type": 0,
+      "value": "Global Policy"
+    }
+  ],
+  "GlobalPolicies.Listing.Table.Header.Actions.Edit": [
+    {
+      "type": 0,
+      "value": "Edit"
+    }
+  ],
+  "GlobalPolicies.Listing.Table.Header.Actions.View": [
+    {
+      "type": 0,
+      "value": "View"
+    }
+  ],
+  "GlobalPolicies.Listing.onboarding.create.new": [
+    {
+      "type": 0,
+      "value": "Let’s get started!"
+    }
+  ],
+  "GlobalPolicies.Listing.onboarding.policies.tooltip": [
+    {
+      "type": 0,
+      "value": "Global Policies provide you the ability to deploy policy mappings to whole gateways and not just one single API. Click below to create your first Global Policy"
```

```suggestion
      "value": "Global policies provide you the ability to deploy policy mappings to all the APIs deployed in a specific gateway and not just one single API. Click below to create your first global policy"
```
Let's change the description if it is suitable.

---

Addressed requested changes by [b341c7d](https://github.com/wso2/apim-apps/pull/515/commits/b341c7d820fee359e17f7a668acbf4ec92d7fcd7)